scripts/update-test-apks.ps1

```diff
@@ -0,0 +1,51 @@
+param([switch] $IfNotExist)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+
+$repoRoot = "$PSScriptRoot/.."
+$apkPrefix = "$repoRoot/test/Sentry.Tests/Internals/android"
+
+function BuildAndroidSample([bool] $UseAssemblyStore, [bool] $UseAssemblyCompression)
+{
+    $sampleDir = "$repoRoot/samples/Sentry.Samples.Android"
+    $framework = 'net6.0-android'
```

Comment:
Does this value need to align with what's inside `Sentry.Samples.Android.csproj`?
I'm thinking of maintaining this, will we need to bump this line when we bump other files/stay in sync?

Reply:
In the end, we may not need this because there's only one target framework, I've removed it from the script.